StudentsManager.Spa/src/pages/HomePage.css

```
@@ -0,0 +1,654 @@
+@import url('https://fonts.googleapis.com/css2?family=Orbitron:wght@400;700;900&family=Roboto+Mono:wght@300;400;500&family=Syncopate:wght@400;700&display=swap');
```

Stylelint (17.7.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Orbitron:wght@400;700;900&family=Roboto+Mono:wght@300;400;500&family=Syncopate:wght@400;700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Orbitron:wght@400;700;900&family=Roboto+Mono:wght@300;400;500&family=Syncopate:wght@400;700&display=swap'" (import-notation)

(import-notation)